labelbox/data/annotation_types/data/raster.py

```diff
@@ -8,10 +8,44 @@
 from google.api_core import retry
 from pydantic import BaseModel
 from pydantic import root_validator
+from joblib import Memory
 
 from .base_data import BaseData
 from ..types import TypedArray
 
+memory = Memory(location="/tmp", verbose=0)
```

Comment:
This prob won't work on windows.

Reply:
Also, I am thinking maybe joblib isn't what we want. I was hoping this would handle a lot of complexity for us. But I think it is easy enough to update the caching logic that already exists to save to disk if users want. Sorry :(

Reply:
and 1 less dependency is nice